cmd/gh2gcs/cmd/root.go

```diff
@@ -136,6 +138,13 @@ func init() {
 		"local directory for releases to be downloaded to",
 	)
 
+	rootCmd.PersistentFlags().BoolVar(
+		&opts.downloadOnly,
+		downloadOnlyFlag,
+		false,
+		"download only the releases, with this flag the files will not be pushed to GCS and you need to set the output-dir flag as well",
```

Comment:
nitpick: how do you feel about this rewording? The current wording felt a little confusing to me, but that could just be me so feel free to ignore suggestion if you don’t see it as an improvement :)

```suggestion
		"only download the releases, do not push them to GCS. Requires the output-dir flag to also be set",
```

Reply:
thanks for your review Daniel!
done